migrations/20231109072809_dmchannel.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE dmchannels (
+    id          text            NOT NULL PRIMARY KEY,
+    from_user   text            NOT NULL,
+    to_user     text            NOT NULL,
+    created_at  timestamp       NOT NULL DEFAULT now(),
+    updated_at  timestamp       NOT NULL DEFAULT now(),
+    permissions json            NOT NULL DEFAULT '{}'::json
```

Comment:
Minor question: DM channels have no permission overrides like regular channels, since they are simply established between two parties. What is this field doing here? Is it for compatibility with some API somewhere?

Reply:
nah we can kill it. I just copied the sql from the channel table

Reply:
Done in https://github.com/zlingapp/server/pull/24/commits/2f75f175c6f82f3d3d1bd99966f68e5ae6bb98e5

Run 
```
cargo sqlx migrate revert
cargo sqlx migrate revert
cargo sqlx migrate run
```
to fix up the schema locally.